docker/docker.bzl

```diff
@@ -20,6 +20,7 @@ load(
     docker_bundle = "container_bundle",
     docker_flatten = "container_flatten",
     docker_image = "container_image",
+    docker_archive = "container_archive",
```

Comment:
We'll want it under `oci/oci.bzl` too.

Reply:
Done.

Reply:
Sorry pressed by accident. It will be done in my next commit.